### PR TITLE
Deploying Angular Node.js Application Using Docker

### PR DESCRIPTION
**Pull request by yzlucas**
Trying to keep this PR simple,once I confirm this works, I should have a second PR to move the ng build and npm install process from the .yml into dockerfile. 
Just want to test this out if it is working fine inside our AWS environment.

### CODE REVIEW

File: client/wfprev-war/backend/node_modules/.bin/mime
Comment (agordon-vivid) [inline client/wfprev-war/backend/node_modules/.bin/mime:1]: I don't think we want to be checking in these node_modules files

**Review by agordon-vivid — CHANGES_REQUESTED** [review]
Let's add a .gitignore enntry for those node_modules directory

File: client/wfprev-war/backend/node_modules/.bin/mime
Comment (dhlevi) [reply client/wfprev-war/backend/node_modules/.bin/mime:1]: We do not!

**Comment by dhlevi** [comment]
Also, might be worth a name change on the client folder. We don't need to add `-war` if it isn't a war anymore

**Comment by yzlucas** [comment]
> Let's add a .gitignore enntry for those node_modules directory

@agordon-vivid  gitignore added. Thanks

**Comment by yzlucas** [comment]
> Also, might be worth a name change on the client folder. We don't need to add `-war` if it isn't a war anymore

Yep, I will add another PR to rename the fodler, and change pathname being used. Just trying to keep this pr clean.

**Review by agordon-vivid — APPROVED** [review]
looks good now.

**Pull request merged**